### ISSUE TITLE
[BUG] App Crashes when player speed is changed

### ISSUE THREAD

**Issue by abhinandval** [jhomlala/betterplayer#156]
**Describe the bug**
Entire app crashed when speed increased during video play.

**To Reproduce**
1. Start video in player
2. Click the overFlowMenuIcon (3dot menu)
3. Select playback speed and pick a speed. 
4. App crashed after this.

**Expected behavior**
Playback speed increased to selected value.

**Flutter doctor**
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 1.22.3, on Microsoft Windows [Version 10.0.19041.572], locale en-IN)
 
[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[!] Android Studio (version 4.1.0)
    X Flutter plugin not installed; this adds Flutter specific functionality.   
    X Dart plugin not installed; this adds Dart specific functionality.
[√] VS Code (version 1.51.1)
[√] Connected device (1 available)  

**Better Player version**
- Version: 0.0.36

**Smartphone (please complete the following information):**
 - Device: Redmi Note 4
 - OS: Android Q (AOSP ROM)

**Additional context**
I've debugged this issue and found this issue to be happening because of accessing a parameter _skipSilence_  while setting the playback speed in `BetterPlayer.java [Line: 281]`. 

![better_player_speed_bug](https://user-images.githubusercontent.com/40170836/100613250-3f6a8f80-333a-11eb-8dd5-4cd5376a05e4.png)

Logcat report. 
```
E/flutter: [ERROR:flutter/shell/platform/android/platform_view_android_jni_impl.cc(43)] java.lang.NoSuchFieldError: No field skipSilence of type Z in class Lcom/google/android/exoplayer2/PlaybackParameters; or its superclasses (declaration of 'com.google.android.exoplayer2.PlaybackParameters' appears in /data/app/com.genskill-IRVqBYw-T281XDgsUKGOfw==/base.apk)
        at com.jhomlala.better_player.BetterPlayer.setSpeed(BetterPlayer.java:280)
        at com.jhomlala.better_player.BetterPlayerPlugin.onMethodCall(BetterPlayerPlugin.java:204)
        at com.jhomlala.better_player.BetterPlayerPlugin.onMethodCall(BetterPlayerPlugin.java:166)
        at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
        at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
        at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:336)
        at android.os.Looper.loop(Looper.java:174)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925)
```

**Comment by padqnn** [comment]
I got the same issue ... 

`E/flutter (20073): [ERROR:flutter/shell/platform/android/platform_view_android_jni_impl.cc(43)] java.lang.NoSuchFieldError: No field skipSilence of type Z in class Lcom/google/android/exoplayer2/PlaybackParameters; or its superclasses (declaration of 'com.google.android.exoplayer2.PlaybackParameters' appears in /data/app/com.splash.app-AbD74Y-V8p6KFfisRNMADg==/base.apk!classes2.dex)
E/flutter (20073): 	at com.jhomlala.better_player.BetterPlayer.setSpeed(BetterPlayer.java:308)
E/flutter (20073): 	at com.jhomlala.better_player.BetterPlayerPlugin.onMethodCall(BetterPlayerPlugin.java:204)
E/flutter (20073): 	at com.jhomlala.better_player.BetterPlayerPlugin.onMethodCall(BetterPlayerPlugin.java:166)
E/flutter (20073): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/flutter (20073): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/flutter (20073): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
E/flutter (20073): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (20073): 	at android.os.MessageQueue.next(MessageQueue.java:336)
E/flutter (20073): 	at android.os.Looper.loop(Looper.java:197)
E/flutter (20073): 	at android.app.ActivityThread.main(ActivityThread.java:8167)
E/flutter (20073): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (20073): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
E/flutter (20073): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
E/flutter (20073):
F/flutter (20073): [FATAL:flutter/shell/platform/android/platform_view_android_jni_impl.cc(942)] Check failed: CheckException(env).
`

**Comment by jhomlala** [comment]
Fixed in 0.0.37.
